lib/frodo/service.rb

```diff
@@ -173,14 +172,15 @@ def logger
     end
 
     def with_metadata?
-      !options.key?(:with_metadata) || options[:with_metadata]
+      !@options.key?(:with_metadata) || @options[:with_metadata]
```

Comment:
Would maybe be easier to understand if the logic were factored as `@options.key?(:with_metadata) && @options[:with_metadata]`

Reply:
ah nevermind-- you want the default behavior-- i get it